web/viewer.css

```diff
@@ -1347,6 +1347,7 @@ html[dir='rtl'] .attachmentsItem > button {
 
 .dialog > .row {
   display: table-row;
+  word-break: break-all;
```

Comment:
Unconditionally adding this rule leads, in my opinion, to decreased readability of the text in the `DocumentProperties` dialog in some cases; see examples below.
Can you please add this rule only for the `chromeFileAccessOverlay`?

---
 - `intelisa.pdf` from the test-suite

![intelisa](https://cloud.githubusercontent.com/assets/2692120/9277819/d6c7fe6e-42ac-11e5-941c-cfd929d418bd.png)

 - http://mirror.switch.ch/ftp/mirror/tex/macros/latex/contrib/biblatex/doc/biblatex.pdf

![biblatex](https://cloud.githubusercontent.com/assets/2692120/9277924/86c8f25a-42ad-11e5-9a7a-fdd8291a1de9.png)